lms/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
@clintonb I think moving the URL into dashboard urls at https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/urls.py make sense.
We should also update the url to be `course_run/{}/refund_status` to be explicit we are getting the refund information from course_run instead of course